src/content/docs/docs/guides/vue.mdx

```diff
@@ -0,0 +1,700 @@
+is logged in: false
+```
+
+## Local Development
+
+For local development, we recommend you use the Bazaar mock server. If you're Vue app is running on the default Vite dev server port, `http://localhost:5173/`, then run the mock server in a new terminal tab:
+
+```bash
+npx @bzr/bazaar-mock -u http://localhost:5173/
+```
+
+You'll be prompted to install the `@bzr/bazaar-mock` NPM package if you haven't done so before.
+
+```bash title="Output"
+Running mock Bazaar server at: http://localhost:3377
+
+App ID is test
+App URL is http://localhost:5173/
+Redirect URIs are http://localhost:5173/
+Use any email to sign up / log in. Code is always 1234
+```
+
+## Authentication
+
+Now that the Bazaar mock server is running, the next step is to authenticate a user.
+
+Add log in and log out buttons:
+
+```vue title="App.vue" ins={4,5}
+<template>
+  <div class="container">
+    <h1>To Do</h1>
+    <button @click="bzr.login">Log in</button>
+    <button @click="bzr.logOut">Log out</button>
+  </div>
+</template>
+```
+
+When you click the button a login pop-up window will open. Log in using the code `1234` and whatever other info you like.
+
+After you login the the pop-up closes, refresh the page and you'll see:
+
+```log title="Browser dev tools console"
+is logged in: true
+```
+
+<Aside>
+  The `bzr.logOut` method deletes an access token from local storage and
+  performs a page refresh.
+</Aside>
+
+Next, refactor the login code so that the login status updates immediately after a successful login, not just on page load.
+
+```vue title="App.vue" del={9} ins={11-21}
+<script setup>
+import { BazaarApp } from "@bzr/bazaar";
+
+const bzr = new BazaarApp({
+  appId: "test",
+  loginRedirectUri: window.location.href,
+});
+
+console.log("is logged in:", bzr.isLoggedIn());
+
+function doLoginActions() {
+  console.log("is logged in:", bzr.isLoggedIn());
+}
+
+// Perform login actions if the user is already logged in on page load
+if (bzr.isLoggedIn()) {
+  doLoginActions();
+}
+
+// Set up handler to perform login actions immediately after a successful login
+bzr.onLogin(doLoginActions);
+</script>
+```
+
+Now, log out and log back in and you'll see `is logged in: true` after the pop-up closes, without the need for a page refresh.
+
+## Display Logged-in Content
+
+Next, you'll display content in your template depending on a user's logged-in status.
+
+```vue title="App.vue" ins={3,7,11,19,21,22,24} del={10}
+<script setup>
+import { BazaarApp } from "@bzr/bazaar";
+import { ref } from "vue";
+
+const bzr = new BazaarApp(/* ... */);
+
+const isLoggedIn = ref(false);
+
+function doLoginActions() {
+  console.log("is logged in:", bzr.isLoggedIn());
+  isLoggedIn.value = true;
+}
+
+// ...
+</script>
+<template>
+  <div class="container">
+    <h1>To Do</h1>
+    <template v-if="!isLoggedIn">
+      <button @click="bzr.login">Log in</button>
+    </template>
+    <template v-else>
+      <button @click="bzr.logOut">Log out</button>
+    </template>
```

Comment:
Why the templates around the buttons? The if statement also works on buttons...

Reply:
Because later more code will be added after the log out button. The template tags hold logged-out code and logged-in code. I could do `v-ifs` on the buttons of course, but would need to show a refactor later.